### PR TITLE
ARXIVNG-4258: dissemination formats from DB metadata

### PR DESCRIPTION
**Pull request by mhl10**

### CODE REVIEW

File: browse/factory.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from arxiv.base.urls import canonical_url, clickthrough_url, urlizer
 from arxiv.base.config import BASE_SERVER
 from arxiv.base import Base
-from arxiv.users.auth import Auth
+# from arxiv.users.auth import Auth
```
Comment (bdc34) [inline browse/factory.py:9]: This commented out due to the "Restore support of login status in Browse" issue.
https://arxiv-org.atlassian.net/browse/ARXIVNG-4354

File: browse/services/documents/db_implementation/db_abs.py [hunk 2]
```diff
@@ -133,25 +123,28 @@ def get_dissemination_formats(self,
...
+                else:
+                    dissemination_formats.append('sciencewise_pdf')
+
+        else:
+            return formats_from_source_type(format_code,
+                                            format_pref,
+                                            False,
+                                            add_sciencewise)
+
         return []
```
Comment (bdc34) [inline browse/services/documents/db_implementation/db_abs.py:148]: Is this return[] extraneous?

**Review by bdc34 — APPROVED** [review]
Seems good.

File: browse/services/documents/db_implementation/db_abs.py (hunk 2 above)
Comment (mhl10) [reply browse/services/documents/db_implementation/db_abs.py]: It is!

**Pull request merged**